java-frontend/src/main/java/org/sonar/java/se/checks/SECheck.java

```diff
@@ -72,15 +74,19 @@ public void scanFile(JavaFileScannerContext context) {
     issues.clear();
   }
 
-  public void reportIssue(Tree tree, String message, Set<List<JavaFileScannerContext.Location>> secondary) {
```

Comment:
I must say that I don't see the point of this refactoring here : if it is a matter of style, well : what is the added value of this effort  ? if there is something else : where is the test ?